test/acceptance/public-api-test.js

```diff
@@ -1475,5 +1475,36 @@ describe('public api', function () {
       expect(result.output).toEqual('\uFEFF<button type="button">LOL, Click me!</button>');
       expect(result.isFixed).toEqual(true);
     });
+
+    it("[.html] files with comments won't cause errors", async function () {
+      // reset config to default value
```

Comment:
I don't think this comment serves any purpose. Can we remove it?

Reply:
I still see this comment 👀

Reply:
Oops, my bad. Removed now.